pyro/distributions/omt_mvn.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+from torch.autograd import Function
+from torch.autograd.function import once_differentiable
+from torch.distributions import constraints
+
+from pyro.distributions.torch.multivariate_normal import MultivariateNormal
+
+
+class OMTMultivariateNormal(MultivariateNormal):
+    """Multivariate normal (Gaussian) distribution with OMT gradients w.r.t. both
+    parameters. Note the gradient computation w.r.t. the Cholesky factor has cost
+    O(D^3), although the resulting gradient variance is generally expected to be lower.
+
+    A distribution over vectors in which all the elements have a joint Gaussian
+    density.
+
+    :param torch.autograd.Variable loc: Mean.
+    :param torch.autograd.Variable scale_tril: Cholesky of Covariance matrix.
+    """
+    params = {"loc": constraints.real, "scale_tril": constraints.lower_triangular}
+    support = constraints.real
```

Comment:
Omit the `.support`, `.reparameterized`, and `.has_rsample` properties; these are already correct.

Reply:
done, thanks